value-macros.h

```diff
@@ -0,0 +1,21 @@
+#ifndef V8GO_VALUE_MACROS_H
+#define V8GO_VALUE_MACROS_H
+
+#define LOCAL_VALUE(val)                   \
```

Comment:
For context-macros.h, you include the files necessary to use the macro. I suggest doing the same thing here.

Reply:
Good point.